README.md

```diff
@@ -1,50 +1,56 @@
 # sysl-template
 
-Simple REST endpoint generation in Go
+Demo REST Application generated using Sysl-Go
 
 ## Prerequisites
 
-- [Sysl v0.11.0 or later ](https://sysl.io/docs/install/)
-- Go 1.13
+- [Sysl v0.140.0 or later ](https://sysl.io/docs/install/)
+- [Arr.ai v0.87.0 or later](https://github.com/arr-ai/arrai)
+- Go 1.14
 
-## How to use this template;
+## Building & Running
 
-- Click the `Use this template` button up the top
-- ctrl-f and replace all instances of `github.com/anz-bank/sysl-template` with your import path, eg `github.com/foobar/myrepo`
-- run `make`
-  - note: `make` must be run with a working internet connection, as it fetches transforms and grammars over the network.
-- run   `go run main.go`
-- congrats! you've just made your first sysl application!
+- run `make gen`. 
+  - This generates the Go code.
+- run `make build`. 
+  - This builds the generated code using the installed Go runtime.
+- run `make run`.
+  - This runs the petdemo app and the downstream apps i.e. flickr and petstore
+- `curl http://localhost:8080/random-pet-pic` or open in browser.
+  - Output: `{"name":"white golden retriever","uri":"http://www.example.com/dog/wgr"}`
 
+Congrats! You've just built and run the sysl-go demo application!
 
 ## Development
-- Refer to the [Makefile](Makefile) to generate client and server code for all of your applications
 
-## File structure 
+- Refer to the [Makefile](Makefile) to generate client and server code for all of your applications.
 
-api/: contains all API specifications for the generated application
+## File structure
 
-gen/: contains all the generated code for the service
+[internal/gen/petdemo]: Contains the all the generated code for the petdemo app.
```

Comment:
You need to provide the link href in parens for it to be a link.

Reply:
Didn't intend to make it a link. The file is present is the same repo.

Reply:
Its a folder.

Reply:
I don't understand why being a folder means it's in square brackets?

Reply:
Well I didn't intend to make it a link, just wanted to highlight it, confused [] with something else. But I do agree that making it a link is a good idea. Will open an issue to track these changes.